xapian-letor/tests/urllib2_file.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
```

Comment:
Is this file actually used? In xapian-core it's used by `submitperftest.in`, but we don't have that in -letor so I think we can kill it :-)